stable/prometheus-operator/values.yaml

```diff
@@ -1260,7 +1260,7 @@ prometheusOperator:
   ##
   image:
     repository: quay.io/coreos/prometheus-operator
-    tag: v0.37.0
+    tag: v0.38.0
```

Comment:
There's a couple of bugfixes in 0.38.1 now

Reply:
Bumping to 0.38.1.
Will also check CRDs